### ISSUE TITLE
libreCAD

### ISSUE THREAD

**Issue by balta1980** [clearlinux/distribution#1983]
Dear people, some weeks ago the libreCAD bundle was added to the software store, I also installed it, but, it seems like that bundle was removed for some reason.

![Captura de pantalla de 2020-05-21 18-36-51](https://user-images.githubusercontent.com/47681659/82612962-31b99d80-9b92-11ea-91c9-d2308d4bcc88.png)
As you can notice there is a missing icon, and when I try to look it in the store it is not shown.

![Captura de pantalla de 2020-05-21 18-40-41](https://user-images.githubusercontent.com/47681659/82613162-a42a7d80-9b92-11ea-8a19-6ddd26cd6e77.png)

Thank you.

BJDS

**Comment by ahkok** [comment]
`sudo swupd bundle-add LibreCAD` - it should still work.

**Comment by balta1980** [comment]
Yes It is truth. I just can not see it in software store. But with swupd it is findable. When I installed it first time, I did it by the software store. Now it is not findable there by me. Thank you anyway.

**Comment by lebensterben** [comment]
>Yes It is truth. I just can not see it in software store. But with swupd it is findable. When I installed it first time, I did it by the software store. Now it is not findable there by me. Thank you anyway.  

@balta1980 So to be precise, the issue is why this bundle is not shown in gnome software.

**Comment by balta1980** [comment]
> > Yes It is truth. I just can not see it in software store. But with swupd it is findable. When I installed it first time, I did it by the software store. Now it is not findable there by me. Thank you anyway.
> 
> @balta1980 So to be precise, the issue is why this bundle is not shown in gnome software.

Yes Sir.
